### ISSUE TITLE
updating notebooks

### ISSUE THREAD

**Issue by djarecka** [dandi/example-notebooks#3]
@bendichter - I'm thinking about updating the notebooks from this repo, but I wonder if your yesterday notebooks were not related to this repo

**Comment by bendichter** [comment]
I just made that on my own, but we could add it to this repo!

**Comment by satra** [comment]
it would be good to update them and perhaps use the nwbstreaming environment directly to pull in the streaming data.

**Comment by bendichter** [comment]
I can work on this

**Comment by satra** [comment]
thanks that looks nice. regarding the search paths, there is a dedicated endpoint for that:

https://dandi.readme.io/reference#dandisets_versions_assets_paths_read

that specific api return will change shortly (https://github.com/dandi/dandi-api/pull/312), but that's what you should use instead of the current formulation.

**Comment by bendichter** [comment]
@satra I'd like to be able to demonstrate API best practices but I don't understand how to use this. What I need is an API where I can input the dandiset identifier and the nwb file path and have it output the asset_id of that NWB file. Is that possible with the API?

**Comment by satra** [comment]
yes that api has exactly those things but will return a list of objects. path is a parameter to that api endpoint. if you fill in the values in readme.io it will show you the corresponding python code.

**Comment by bendichter** [comment]
@satra 

I've been playing around with it and this is the closest I've gotten

![image](https://user-images.githubusercontent.com/844306/121752095-6e2da380-cadd-11eb-837f-4d6489da4f5d.png)

but this neither returns the single object I am looking for nor returns the asset ID. Also, the documentations says:

> The specified path must be a folder; it either must end in a slash or (to refer to the root folder) must be the empty string.

but I want to specify the path of a file.

**Comment by satra** [comment]
really sorry, i gave the wrong api endpoint.

```
import requests

url = "https://api.dandiarchive.org/api/dandisets/000053/versions/draft/assets/"

querystring = {"path":"sub-npI1"}

headers = {"Accept": "application/json"}

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)
```

**Comment by satra** [comment]
and you should be able to give the fullpath. the other endpoint returns the list of objects at a given level in the hierarchy

**Comment by bendichter** [comment]
that works, thanks @satra
